src/commands/scanner/run/dfa.ts

```diff
@@ -69,7 +69,17 @@ export default class Dfa extends ScannerRunCommand {
 			summary: getMessage(BundleName.RunDfa, 'flags.pathexplimitSummary'),
 			description: getMessage(BundleName.RunDfa, 'flags.pathexplimitDescription'),
 			env: 'SFGE_PATH_EXPANSION_LIMIT'
-		})
+		}),
+		'enablecaching': Flags.boolean({
+			summary: '',
+			description: '',
+			env: 'SFGE_ENABLE_CACHING'
+		}),
+		'cachepath': Flags.string({
+			summary: '',
+			description: '',
+			env: 'SFGE_FILES_TO_ENTRIES_CACHE_LOCATION'
+		}),
```

Comment:
Question: Am I remembering correctly that delta runs are primarily a VSCode extension thing? If that's true, then is there any use case for these flags outside of VSCode, and should we consider using the `hidden: true` property to hide those flags from the help text?